src/css/base.css

```diff
@@ -24,8 +26,10 @@ body {
   display: flex;
   flex-direction: column;
   font-size: 14px;
+  line-height: 24px;

```

Comment:
Line height should be set as a float number

Reply:
Here I had a discussion even when we were doing homework. I don’t like the values like 171.43% or 1.72. They end up giving imprecise sizes. Relative Line Height = (24px / 14px) * 100%
                      ≈ 171.43%

Reply:
Better to be consistent as there is already relative line height used in other places.